.github/workflows/simorgh-create-github-release.yml

```diff
@@ -29,6 +29,6 @@ jobs:
           tag_name: ${{ steps.release_tag.outputs.new_tag }}
           release_name: Release ${{ steps.release_tag.outputs.new_tag }}
           body: |
-            ${{ github.event.commits[0].message }}
-          draft: false
+            ${{ github.event.head_commit.message }}
```

Comment:
`head_commit` is the commit when you merge the branch to latest e.g. in https://github.com/bbc/simorgh/commit/3b7dfcb0614b9cc88b992c004094d92fc7443206, it's:
```
 Merge pull request #8298 from bbc/github-release-notes

Use merge commit as release notes for release created by GitHub actions
```